pkg/web/oauthclient/cookie_state.go

```diff
@@ -41,7 +41,7 @@ func (oc *OAuthClient) StateCookie() *cookie.Cookie {
 		HTTPOnly: true,
 		Path:     oc.getMountPath(),
 		MaxAge:   10 * time.Minute,
-		SameSite: http.SameSiteNoneMode,
```

Comment:
Is this intended ?

Reply:
Oh good catch, no that's not intended. This sneaked in somehow.